src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input } from '@angular/core';
+
+export const SPINNER_BASE_CLASS = 'spinner';
+
+export const SPINNER_INVERSE = 'spinner-inverse';
+export const SPINNER_INLINE = 'spinner-inline';
+
+export const SPINNER_SMALL_SIZE = 'spinner-sm';
+export const SPINNER_MEDIUM_SIZE = 'spinner-md';
+export const SPINNER_LARGE_SIZE = 'spinner-lg';
+
+@Component({
+  selector: 'clr-spinner',
+  template: `<ng-content></ng-content>`,
+  host: {
+    '[attr.class]': 'assignClass',
+    '[attr.aria-live]': 'setAriaLive',
+    '[attr.aria-busy]': 'true',
+  },
+})
+export class ClrSpinner {
+  // Style
+  @Input('inline') inline: string;
+  @Input('inverse') inverse: string;
+
+  // Size
+  @Input('small') small: string;
```

Comment:
nit: you don't need the inner string in the decorator if the property name is the same,
`@Input() small: boolean`